src/battle_util.c

```diff
@@ -1463,7 +1463,9 @@ void CancelMultiTurnMoves(u8 battler)
                         
                         // If this CancelMultiTurnMoves is occuring due to attackcanceller or VARIOUS_GRAVITY_ON_AIRBORNE_MONS
                         if (gBattlescriptCurrInstr[0] == 0x0 || (gBattlescriptCurrInstr[0] == 0x76 && gBattlescriptCurrInstr[2] == 76))
+                        {
                             gBattleStruct->skyDropTargets[0] = 0xFE - battler;
+                        }
```

Comment:
I can't select all the lines involved, but the `else if` statements below this `if` should all come one after the other, that is without empty spaces inbetween, in order to keep consistency with upstream *(Pret)*.